nmostesting/GenericTest.py

```diff
@@ -383,6 +397,97 @@ def validate_schema(self, payload, schema):
     def do_request(self, method, url, **kwargs):
         return TestHelper.do_request(method=method, url=url, **kwargs)
 
+    def make_key_cert_files(self, cert_file, key_file):
```

Comment:
Would it be worth adding a few more comments to this function, explaining which of the parameters are fixed e.g. to meet spec guidance, and which are free choice?

Reply:
More parameters will be passed in i.e. less hardcoded in the function.

Reply:
Fixed